OracleWebCenterPortal/dockerfiles/12.2.1.4/README.md

```diff
@@ -1,6 +1,24 @@
 # Oracle WebCenter Portal 12.2.1.4.0 on Docker
+
+## Mount a host directory as a data volume
+ The default location of the volume in container is under /var/lib/docker/volumes. There is an option to mount a directory from the host into a container as volume. In this project we will use that option for the data volume.
+To mount a host directory `/scratch/wcpdocker/volumes/wcpportal`(`DATA_VOLUME`) as a data volume, execute the below command.
+
+> The userid can be anything but it must belong to uid:guid as 1000:1000, which is same as 'oracle' user running in the container.
+
+> This ensures 'oracle' user has access to shared volume.
+
+```
+$ sudo mkdir -p /scratch/wcpdocker/volumes/wcpportal
+$ sudo chown 1000:1000 /scratch/wcpdocker/volumes/wcpportal
+```
+  All container operations are performed as **'oracle'** user.
+  
+  **Note**: If a user already exist with **'-u 1000 -g 1000'** then use the same user. Or modify any existing user to have uid-gid as **'-u 1000 -g 1000'**
+
+ 
 ##  Preparing to Run Oracle WebCenter Portal Docker Container
```

Comment:
Please remove the word "Docker" from this heading. It's also missing the word "the", i.e. "Preparing to run _the_ Oracle WebCenter Portal container", assuming there is only one container. If there are multiple containers, then perhaps "Preparing to run the Oracle WebCenter Portal containers"